src/utils/format-unit-size.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Replaces Petabox www/common/Util::humanSize()
+ */
+import { nothing } from 'lit';
+import type { Divisor } from './format-count';
+
+export function formatUnitSize(
+  size: number | undefined,
+  ndecimals: Divisor,
+  seprator: String = ' '
+) {
+  let times = 0;
+  let itemSize = size;
+  const timesNames = [];
+
+  if (itemSize === undefined) return nothing;
+
+  while (itemSize > 1024) {
+    itemSize /= 1024;
+    times += 1;
+  }
+
+  timesNames[0] = 'bytes';
+  timesNames[1] = 'kilobytes';
+  timesNames[2] = 'megabytes';
+  timesNames[3] = 'gigabytes';
+  timesNames[4] = 'terabytes';
+  timesNames[5] = 'petabytes';
+  timesNames[6] = 'exabytes';
+  timesNames[7] = 'zettabytes';
+  timesNames[8] = 'yottabytes';
+
+  itemSize = Math.round(itemSize * ndecimals) / ndecimals;
```

Comment:
The naming of `ndecimals` implies it should accept the desired number of decimal places (simple to understand). But here it seems to actually require the corresponding power of 10, not the number of decimal places? I think we should go with the former case, which is less likely to cause confusion.

Suggestion: Change the type of `ndecimals` to `number`, and replace this line with
```suggestion
  const magnitude = Math.pow(10, ndecimals);
  itemSize = Math.round(itemSize * magnitude) / magnitude;
```